projects/alphaX-protocol/index.js

```diff
@@ -3,10 +3,18 @@ const { sumTokensExport } = require('../helper/unwrapLPs');
 
 const tokens = [
   ADDRESSES.ethereum.USDT,
+  ADDRESSES.arbitrum.USDT
 ];
 
+const owners = [
+  '0x95e2A6458419b7A38193CB853F45fD7329443A90',
+  '0x70eF5649c0b51162C62CE38d807b8aD61Ac2560d',
+
+];
+
+
 const walletAddresses = {
- ethereum: ['0xA61a6E696B7C566DA42B80dA27d96e7104bcec99'],
+ ethereum: ['0xA61a6E696B7C566DA42B80dA27d96e7104bcec99','0x95e2A6458419b7A38193CB853F45fD7329443A90'],
```

Comment:
what is this gnosis wallet and why are you counting it under tvl?

Reply:
It holds both the user funds from our previous centralized exchange (CEX) and our insurance fund. Since we were previously CEX, there are still historical users and remaining funds.  We're gradually migrating them to DEX.  For the insurance fund, it is reserved to compensate DEX users in the event of losses due to hacking or system errors."

Reply:
hmm, then we could track this as treasury but not as tvl of dex since it was not deposited by dex users. Can I make that change now?

Reply:
Sure. Need I update the script?  How to fix it if so?